rootfs/templates/wrapper

```diff
@@ -46,6 +46,10 @@ function debug() {
 	fi
 }
 
+function run_exit_hooks() {
+	command -v on_geodesic_exit >/dev/null && on_geodesic_exit
```

Comment:
```suggestion
	command -v geodesic.exit >/dev/null && geodesic.exit
```
Perhaps use `geodesic.$hook`

Reply:
Are you sure? This is going to need to be installed on the user's `$PATH` on their host computer.

Reply:
This is a command that the user can optionally have installed on their machine, right? Then a prefix of `geodesic` makes more sense to me, rather than a prefix of `on_`. 

We can maybe do `geodesic.on_exit` or something like that?

Reply:
OK, but no dots. Dots have way too much significance. I like to blame DOS but it really started as early as Unix and C.